src/Microsoft.Data.Analysis/DateTimeComputation.cs

```diff
@@ -0,0 +1,310 @@
+﻿using System;
```

Comment:
This will need the license header. Copying from another file should work just fine